lib/modules/exec.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (dep) {
+  const { shell, console, colors } = dep
```

Comment:
I don't see any point of injecting modules like that, it makes it a bit unreadable.
I'd just require chalk directly.